doc/deployment/amazon_web_services/aws-deploy-pachyderm.md

```diff
@@ -0,0 +1,238 @@
+# Deploy Pachyderm on AWS
+
+After you deploy Kubernetes cluster by using `kops` or `eksctl`,
+you can then deploy Pachyderm on top of that cluster.
+
+You need to complete the following steps to deploy Pachyderm:
+
+1. Deploy `pachctl` as described in [Deploy pachctl](../../../getting_started/local_installation.html#pachctl).
+1. Add stateful storage for Pachyderm as described in [Add Stateful Storage](#add-stateful-storage).
+1. Deploy Pachyderm by using an [IAM Role](deploying-with-an-iam-role)
+(recommended) or [Static Credentials](deploying-with-static-credentials).
+
+## Add Stateful Storage
+
+In Amazon, stateful data is stored in S3 buckets that reside on Elastic
```

Comment:
```
In Amazon, stateful data is stored in S3 buckets that reside on Elastic
Block Storage (EBS) persistent volumes. Pachyderm requires an object
store for the PFS metadata.
```

This is incorrect. S3 is a distinctly different thing than EBS volumes (PVs). Volumes and object store are not the same. Also note: both data and some metadata (e.g. hashtrees) are stored in the object store bucket, we generally describe etcd as the metadata which needs the PV. Also, it's not just PFS metadata, it's both PFS and PSS so we should just call it Pachyderm metadata. 

Change this whole paragraph to:
```suggestion
Pachyderm requires two pieces of persistent storage:  an object
store (S3) bucket for your data and an EBS persistent volume (PV) that is used to store Pachyderm metadata.```